src/content/roadmap/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Ethereum roadmap
+description: An overview of Ethereum's upgrade plans
+lang: en
+template: staking
+emoji: ":money_with_wings:"
+image: ../../../assets/staking/leslie-pool.png
+alt: Leslie the rhino swimming in the pool.
+sidebarDepth: 2
+summaryPoints:
+  - Learn about how Ethereum will evolve into the future
+  - Read why Ethereum is still being worked on
+  - Understand how Ethereum is setting up for long term success
+---
+
+Ethereum is a powerful platform for global coordination, but it is not finished! An ambitious set of improvements will upgrade Ethereum from its current form into a fully scaled, maximally resilient platform. These upgrades are laid out in the Ethereum roadmap.
+
+**To learn about previous upgrades to Ethereum, such as EIP-1559 or The Merge, please visit our [History](/history/) page**
+
+## What changes are coming to Ethereum? {#what-changes-are-coming}
+
+The Ethereum roadmap outlines the specific improvements that will be made to protocol in the future. Overall, the roadmap will bring the following benefits to Ethereum users:
+
+<CardGrid>
+  <Card title="Faster, cheaper transactions" emoji="🐟">
+
+    Today's rollups are still in their "training wheels" phase where critical components are run by centralized operators. Decentralizing the rollup infrastructure is a critical part of scaling Ethereum without compromising on Ethereum's core ethos of decentralization. At the same time, it is too expensive for rollups to post their data to Ethereum and this limits how cheaply users can transact. Adding data-blobs to blocks and sharding data across nodes will solve this problem, leading to near-instant, <$0.001 transactions for users.
+
+    [Read how Ethereum will scale](/roadmap/faster-transactions/)
```

Comment:
Would it make sense to call this page `/roadmap/scaling/` vs. `/roadmap/faster-transactions/`?

Reply:
Haven't looked but I suspect search volume around "Ethereum scaling" is higher than terms around faster transactions.

Reply:
done!